Countries/Country.swift

```diff
@@ -7,15 +7,15 @@
 
 import Foundation
 
-struct Country: Decodable {
-//    var code: UUID
-    var name: String
-    var capital: String
-    var continent: Continent
-    var emoji: String
-}
-
-struct Continent: Decodable {
-    var name: String
-//    var countries: [Country]
-}
+//struct Country: Decodable {
+////    var code: UUID
+//    var name: String
+//    var capital: String
+//    var continent: Continent
+//    var emoji: String
+//}
+//
+//struct Continent: Decodable {
+//    var name: String
+////    var countries: [Country]
+//}
```

Comment:
Should it file still be in code base if it all commented?

Reply:
Ok, you're right I will delete it now)